### ISSUE TITLE
Unclear whether inshellisense works only in the current directory

### ISSUE THREAD

**Issue by adder32** [microsoft/inshellisense#5]
**Describe the bug**
Unclear whether inshellisense works only in the current directory

**To Reproduce**
```sh
alex@smartalex-lap:~$ cd Repositories/
alex@smartalex-lap:~/Repositories$ ls
chronos  godbolt-reaper  linked_list_py  porcu-regex  recursivedescent  tmp  ttt  vivify  wrath
alex@smartalex-lap:~/Repositories$ is --shell bash
> cd wrath/
> ls
chronos  godbolt-reaper  linked_list_py  porcu-regex  recursivedescent	tmp  ttt  vivify  wrath
> pwd
/home/alex/Repositories
```

**Expected behavior**
By running the commands above you should end up in `/home/alex/Repositories/wrath`.

**Environment**
 - OS: `XUbuntu 22.04.1`
 - Output of `inshellisense --version`: `0.0.1-rc.1`
 - Nodejs Version: `v20.6.1`

**Comment by philibertperusse** [comment]
Same issue using Powershell, 'cd' command seems to have no effect.

**Comment by Roconda** [comment]
Same here, didn't work on MacOS with Fish shell as well. Node version 20.

**Comment by cpendery** [comment]
This was originally intentional since I didn't expect an inshellisense session to last longer than a few commands in the current directory

**Comment by abulka** [comment]
Can't autocomplete references to filenames in other directories e.g. I type
```
ls ~/.zsh
```
then hit `TAB` to autocomplete (which should list e.g.
```
.zsh_history    .zsh_sessions/  .zshrc
```
but instead I get nothing - no completion of filenames at all.

**Comment by cpendery** [comment]
Working on addressing all these issues of respecting the underlying shell

**Comment by cpendery** [comment]
Closing as this was fixed in [0.0.1-rc.5](https://github.com/microsoft/inshellisense/releases/tag/0.0.1-rc.5)
